src/serde/deserialize_program.rs

```diff
@@ -64,6 +64,16 @@ pub struct Identifier {
     #[serde(default)]
     #[serde(deserialize_with = "bigint_from_number")]
     pub value: Option<BigInt>,
+
+    pub full_name: Option<String>,
+    pub members: Option<HashMap<String, Member>>,
+}
+
+#[derive(Clone, Debug, Deserialize, PartialEq)]
+pub struct Member {
+    pub cairo_type: Option<String>,
+    #[serde(deserialize_with = "bigint_from_number")]
+    pub offset: Option<BigInt>,
```

Comment:
Is it necessary for this to be a BigInt? shouldn't it be enough with a usize?

Reply:
I'm not sure. Since Python's `BigInt` is just `int`, it's difficult to differentiate between the two.

However, now that you mention it, a relocatable's address is just `usize`, therefore you're probably right in that it should be usize.